merlin/models/tf/blocks/core/combinators.py

```diff
@@ -209,19 +209,19 @@ def call(self, inputs, training=False, **kwargs):
 
         outputs = inputs
         for i, layer in enumerate(self.layers):
-            if i == len(self.layers) - 1:
-                filtered_kwargs = filter_kwargs(kwargs, layer, filter_positional_or_keyword=False)
-                filtered_kwargs.update(
-                    filter_kwargs(
-                        {**dict(training=training), **kwargs},
-                        layer.call,
-                        filter_positional_or_keyword=False,
-                    )
-                )
-            else:
-                filtered_kwargs = filter_kwargs(
-                    dict(training=training), layer, filter_positional_or_keyword=False
+            filtered_kwargs = filter_kwargs(
+                {**dict(training=training), **kwargs},
+                layer,
+                cascade_kwargs_if_possible=True,
+            )
+            filtered_kwargs.update(
+                filter_kwargs(
+                    {**dict(training=training), **kwargs},
```

Comment:
```suggestion
                    {training=training, **kwargs},
```

Reply:
Thanks, just added your suggestion in my latest commit